File: app/src/main/java/com/example/qpony/Network/ApiUtils.java
package com.example.qpony.Network;

import android.content.Context;

public class ApiUtils {

    private static final String BASE_URL = "https://adrian.kubahaha.tk/";


    private ApiUtils() {
    }


    public static APIService getAPIService(Context context) {
        return RetrofitClient.getClient(BASE_URL, context).create(APIService.class);
    }


}

File: app/src/main/java/com/example/qpony/Network/ApiUtils.kt
package com.example.qpony.Network

import android.content.Context

object ApiUtils {

    private val BASE_URL = "https://adrian.kubahaha.tk/"


    fun getAPIService(context: Context): APIService {
        return RetrofitClient.getClient(BASE_URL, context).create(APIService::class.java)
    }


}